<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CartWidget</class>
 <widget class="QWidget" name="CartWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <widget class="QFrame" name="frame">
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout" stretch="1,9">
      <item>
       <layout class="QHBoxLayout" name="appBarLayout"/>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="6,4">
        <item>
         <layout class="QVBoxLayout" name="cartItemsLayout"/>
        </item>
        <item>
         <layout class="QVBoxLayout" name="orderSummaryLayout"/>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
